### ISSUE TITLE
possible typo?

### ISSUE THREAD

**Issue by shawei3000** [ewrfcas/Reinforced-Mnemonic-Reader#1]
# fusion
        hh = tf.concat([h, h_B, h * h_B, h - h_B], axis=-1)
        x = tf.nn.relu(conv1d(uv, filters, 1, name='Wr'))           # (jim sha) should be hh?
        g = tf.nn.sigmoid(conv1d(hh, filters, 1, name='Wg'))

**Comment by ewrfcas** [comment]
@shawei3000  thanks for your correction

**Comment by shawei3000** [comment]
let me know if performance increases. with elmo, has potential to reach 84+ (FM)

**Comment by woshiyyya** [comment]
@ewrfcas Wow, did you write this model according to the paper by yourself? Or you are the author?

**Comment by ewrfcas** [comment]
@woshiyyya It is not a official version. I find that the earlier version of the paper can achieve better performance in the pytorch implement. Maybe I will verify the earlier version recently.

**Comment by doris-he** [comment]
Error: [E050] Can't find model 'en'. It doesn't seem to be a shortcut link, a Python package or a valid path to a data directory.
preprocess.ipny

**Comment by ewrfcas** [comment]
@doris-he , this problem maybe caused by missing the spacy language package?
